### PR TITLE
Build updates for Spark 1.4.1

### PR DESCRIPTION
**Pull request by allenday**
These are the changes needed to get this to build against stock Spark 1.4.1

### CODE REVIEW

File: bin/build-and-run [hunk 1]
```diff
@@ -17,7 +17,8 @@ import time
 
 # The express-D project directory
 proj_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
-sbt_cmd = "sbt/sbt"
+#sbt_cmd = "sbt/sbt"
```
Comment (adarob) [inline bin/build-and-run]: Remove line.

File: src/main/scala/expressd/ExpressD.scala [hunk 2]
```diff
@@ -1067,7 +1074,13 @@ object ExpressD {
 
       // ===== Debugging =====
       // Print out some memory usage info.
-      var accumFragmentsSize = sc.accumulable(0.0)
+      //var lap = new LongAccumulatorParam()
```
Comment (adarob) [inline src/main/scala/expressd/ExpressD.scala]: Do we need these commented lines?

**Comment by allenday** [comment]
updated, thx

**Comment by adarob** [comment]
Thanks!

**Pull request merged**